dotnet/src/SemanticKernel/Planning/SkillPlan.cs

```diff
@@ -8,7 +8,7 @@ namespace Microsoft.SemanticKernel.Planning;
 /// <summary>
 /// Object that contains details about a plan and its goal and details of its execution.
 /// </summary>
-public class Plan
+public class SkillPlan
```

Comment:
not sure about the name, we have IPlan, PlanStep and SkillPlan and there's a lot of overlap and duplication, the names are also very hard to understand IMO

Reply:
SkillPlan is going to be deleted basically, and I wanted to clear the way for possibly having a new `Plan: IPlan` -- but not even sure if that will happen.